### PR TITLE
Proposal for a default scenario file,

### PR DESCRIPTION
**Pull request by jsuchome**
to be used with cloud6 and 2 nodes setup.

Based on this one, we could create differen scenarios that can be used to cover different cases

EDIT: to be called as

``export want_node_aliases=controller=1,compute-kvm=1 nodenumber=2 scenario='cloud6-default.yml' mkcloud instonly setup_aliases batch``

### CODE REVIEW

**Comment by jsuchome** [comment]
Ouch, it has c13.cloud.suse.de as part of node names, so this couldn't work...

**Comment by MaximilianMeister** [comment]
i think with an `.erb` template we could be more flexible with the node names. but that would also need a script that compiles it to `yaml`

**Comment by jsuchome** [comment]
@MaximilianMeister maybe. Let's wait for the result of https://trello.com/c/y8UlTUq3/448-5-p4-mkcloud-research-how-to-design-scenario-file-support

Specifically, this needs some solution for assigning the aliases to nodes. May be done by mkcloud automatically, but better way could be to have some other config file for that.

**Comment by jsuchome** [comment]
Tested mkcloud with this, and it seems to work fine (there are some problems with crowbar_batch, but not related to yml file itself)

**Comment by vuntz** [comment]
+1; but not sure if we want to get the PR for the aliases in first?

**Comment by nkrinner** [comment]
+1

**Comment by vuntz** [comment]
@jsuchome could you remove the trailing `,` from the commit message?

File: scripts/scenarios/cloud6/cloud6-2nodes-default.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  attributes:
+  deployment:
+    elements:
+      keystone-server:
+      - @@controller@@
+- barclamp: swift
+  attributes:
+    zones: 1
+    keystone_delay_auth_decision: true
+    allow_versions: true
```
Comment (vuntz) [inline scripts/scenarios/cloud6/cloud6-2nodes-default.yml]: Reviewing again: I'd feel more comfortable if we could explicitly set some more attributes here, even if they're already set correctly in the template:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L2247

(these are required for tempest tests)

Comment (jsuchome) [inline scripts/scenarios/cloud6/cloud6-2nodes-default.yml]: Adapted, tested with simple crowbar_batch call, now testing with full mkcloud run...

**Comment by vuntz** [comment]
+1

File: scripts/scenarios/cloud6/cloud6-2nodes-default.yml (hunk 1 above)
Comment (jsuchome) [inline scripts/scenarios/cloud6/cloud6-2nodes-default.yml]: ... which ended with correct deployment

**Pull request merged**